src/main/cc/proxy_client/proxy_client.cc

```diff
@@ -200,8 +204,39 @@ int ComputeInputs(int argc, char** argv, const char** env, const string& cwd, co
       // Fall back on computing from the command, but warn.
```

Comment:
I guess you could set use_args_inputs to true here (and down below in header-abi-dumper block) and then in line 239, you can simply do:
```
if (use_args_inputs) {...}
```

 instead of 
```
if (inputs.empty() || use_args_inputs) {...}
```